src/metatensor/models/cli/eval.py

```diff
@@ -176,7 +176,8 @@ def _eval_targets(
     rmse_values = rmse_accumulator.finalize(not_per_atom=["positions_gradients"])
     # print the RMSEs with MetricLogger
     metric_logger = MetricLogger(
-        model_capabilities=model.capabilities(),
+        logobj=logger,
+        model_outputs=_get_outputs(model),
```

Comment:
We should mention in the docstring that non-exported models can only use this function if they define a `.outputs`

Reply:
Doesn't `eval` only works for exported models?

Reply:
> Doesn't eval only works for exported models?

Yes and no. The public function yes but we use the internal one also to evaluate the model during training.

> We should mention in the docstring that non-exported models can only use this function if they define a .outputs

Agreeing with Guillaume if this is the public one we check that models are already exported.

Reply:
Whoops, sorry, I commented on the wrong piece of code. Of course the comment would have to go in the docstring of our internal `eval`